src/main/java/net/rptools/maptool/client/ui/zone/ZoneView.java

```diff
@@ -347,14 +347,21 @@ private List<ContributedLight> calculateLitAreaForLightSource(
     }
     lightSourceArea.transform(translateTransform);
 
-    final var lightSourceVisibleArea =
-        FogUtil.calculateVisibility(
-            p,
-            lightSourceArea,
-            getTopologyTree(Zone.TopologyType.WALL_VBL),
-            getTopologyTree(Zone.TopologyType.HILL_VBL),
-            getTopologyTree(Zone.TopologyType.PIT_VBL),
-            getTopologyTree(Zone.TopologyType.COVER_VBL));
+    Area lightSourceVisibleArea = null;
+
+    if (lightSource.isPassThroughVBL()) {
+      lightSourceVisibleArea =
+          FogUtil.calculateVisibility(p, lightSourceArea, null, null, null, null);
```

Comment:
This call can be removed as it will just return a copy of `lightSourceArea` back again.